src/test/TxTests.cpp

```diff
@@ -336,6 +336,24 @@ validateTxResults(TransactionFramePtr const& tx, Application& app,
 TxSetResultMeta
 closeLedgerOn(Application& app, uint32 ledgerSeq, int day, int month, int year,
               std::vector<TransactionFrameBasePtr> const& txs)
+{
+    return closeLedgerOn(app, ledgerSeq, day, month, year, 0 /* hour */,
+                         0 /* minute */, 0 /* second */, txs);
+}
+
+TxSetResultMeta
+closeLedgerOn(Application& app, uint32 ledgerSeq, int day, int month, int year,
```

Comment:
Similarly here maybe use default args (and/or change the arg to time_t and make callers write a call to `getTestDateTime`)?

Reply:
Default args now used here too.